src/libraries/System.Net.ServerSentEvents/src/System/Net/ServerSentEvents/SseItem.cs

```diff
@@ -9,12 +9,16 @@ public readonly struct SseItem<T>
     {
         /// <summary>The event's type.</summary>
         internal readonly string? _eventType;
+        /// <summary>The event's id.</summary>
+        private readonly string? _eventId;
 
         /// <summary>Initializes the server-sent event.</summary>
         /// <param name="data">The event's payload.</param>
         /// <param name="eventType">The event's type.</param>
-        public SseItem(T data, string? eventType)
+        public SseItem(T data, string? eventType = null)
         {
+            Helpers.ValidateParameterDoesNotContainLineBreaks(eventType, nameof(eventType));
```

Comment:
Can you add an XML comment for the exception?

Reply:
Done.